.gitignore

```diff
@@ -27,3 +27,8 @@
 /package-lock.json
 /pnpm-lock.yaml
 /yarn.lock
+
+# Now
+.now
+.env.build
+.env
```

Comment:
Use slashes here. See comment at the top of the file for why!

```suggestion
# Now
/.now
/.env.build
/.env
```

also newline at end, please